website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -518,21 +521,21 @@ This POST endpoint requires a JSON object with the following properties as a req
-`data.relationships.project.data.type`                  | string  |           | Must be `"project"`.
-`data.relationships.vars.data[].type`                   | string  |           | Must be `"vars"`.                                                                                                                                                                                                     |
-`data.relationships.vars.data[].attributes.key`         | string  |           | The name of the variable.                                                                                                                                                                                             |
-`data.relationships.vars.data[].attributes.value`       | string  | `""`      | The value of the variable.                                                                                                                                                                                            |
-`data.relationships.vars.data[].attributes.description` | string  |           | The description of the variable.                                                                                                                                                                                      |
-`data.relationships.vars.data[].attributes.category`    | string  |           | Whether this is a Terraform or environment variable. Valid values are `"terraform"` or `"env"`.                                                                                                                       |
-`data.relationships.vars.data[].attributes.hcl`         | boolean | `false`   | Whether to evaluate the value of the variable as a string of HCL code. Has no effect for environment variables.                                                                                                       |
-`data.relationships.vars.data[].attributes.sensitive`   | boolean | `false`   | Whether the value is sensitive. If true then the variable is written once and not visible thereafter.                                                                                                                 |
+| Key path | Type   | Default | Description |
+| --- | --- | --- | ---  |
+| `data.type`                                             | string  |           | Must be `"workspaces"`. |
+| `data.attributes.name`                                  | string  |           | The name of the workspace, which can only include letters, numbers, -, and _. This will be used as an identifier and must be unique in the organization. |
+| `data.attributes.description`                           | string  | (nothing) | A description for the workspace. |
+| `data.attributes.auto_apply`                            | boolean |  `false`  | If `true`, HCP Terraform will automatically apply changes when a Terraform plan is successful, with some exceptions. |
```

Comment:
```suggestion
| `data.attributes.auto_apply`                            | boolean |  `false`  | If `true`, HCP Terraform automatically applies changes when a Terraform plan is successful. |
```
Unless we can describe the exceptions or link to at least one example, I can see this being frustrating for users.